src/app/events/tasks.ts

```diff
@@ -113,7 +113,7 @@ const onlyDriverRules: FieldOptions<Task, string> = {
       return Boolean(t?.eventBelongToOrgUser([Roles.trainee, Roles.dispatcher]))
     return Boolean(t?.eventBelongToOrgUser(Roles.dispatcher))
   },
-  allowApiRead: Allow.authenticated,
+  allowApiRead: true, // TODO WARNING: Audit this. I've added an apiPrefilter.
```

Comment:
ok

Reply:
![image](https://github.com/noam-honig/trempim/assets/16635859/55ae66a4-5edd-4f14-827d-6cd7f7a27b47)

Reply:
Thanks

Reply:
@noam-honig Now I get 'אינך מורשה פעולה זו'. I see that such an error occurs when 403 is returned but it's unclear to me where exactly the 403 is coming from.

I thought it might be CanSeeUsersGuard but even temporarily turning that to always true to test doesn't fix the issue. I'm guessing it's some piece of code manually set somewhere in low level API logic. But did not find anything in task.ts or api.ts

Reply:
@noam-honig Never mind, found it. allowApiRead on the User entity.